registry.go

```
@@ -79,7 +81,12 @@ func (r *Registry) startWatcher(resource Resource) {
 	if _, ok := r.watchers[resource]; ok {
 		return
 	}
-	collection := r.gobserve.DB.C(resource.Collection)
+
+	collectionName := r.dataMapping.KindToCollection[resource.Kind]
+	if collectionName == "" {
+		log.Println("error " + resource.Kind + "not supported")
```

@sfriedel we might need some error handling here, otherwise it's a silent fail